features/step_definitions/attachment_steps.rb

```diff
@@ -63,12 +63,11 @@
 end
 
 Then(/^the attachments should be in the following order:$/) do |attachment_list|
```

Comment:
Once the above is reverted you can test the output of both by using Ollies new helper `using_design_system?` added here https://github.com/alphagov/whitehall/pull/6953

```
if using_design_system?
  attachment_names = all("table td:first").map(&:text)

  attachment_list.hashes.each_with_index do |attachment_info, index|
    attachment = Attachment.find_by(title: attachment_info[:title])
    expect(attachment.title).to eq(attachment_names[index])
  end
else
   attachment_ids = all(".existing-attachments > li").map { |element| element[:id] }
  
   attachment_list.hashes.each_with_index do |attachment_info, index|
     attachment = Attachment.find_by(title: attachment_info[:title])
     expect("attachment_#{attachment.id}").to eq(attachment_ids[index])
  end
end  
```

Reply:
Done, thank you @davidgisbey 👍